nautobot/dcim/choices.py

```diff
@@ -930,6 +931,7 @@ class InterfaceTypeChoices(ChoiceSet):
                 (TYPE_VIRTUAL, "Virtual"),
                 (TYPE_BRIDGE, "Bridge"),
                 (TYPE_LAG, "Link Aggregation Group (LAG)"),
+                (TYPE_TUNNEL, "Tunnel"),
```

Comment:
New interface type to allow better filtering for for `tunnel_interface` in `VPNTunnelEndpoint` model.